Crash with lazy_init set

```bash
│ Error: Plugin did not respond
│
│   with module.rode.rode_policy_group.rode_demo,
│   on ../modules/rode/policy.tf line 29, in resource "rode_policy_group" "rode_demo":
│   29: resource "rode_policy_group" "rode_demo" {
│
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more
│ details.
╵

Stack trace from the terraform-provider-rode_v0.1.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x68 pc=0x16e6fdf]

goroutine 66 [running]:
github.com/alexashley/terraform-provider-rode/internal/provider.resourcePolicyRead(0x19e8ca8, 0xc0008a05a0, 0xc00033c480, 0x18d8f20, 0xc0004b60c0, 0xc000749660, 0xc0004dd908, 0x100df38)
	github.com/alexashley/terraform-provider-rode/internal/provider/resource_rode_policy.go:129 +0x1df
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc0003800e0, 0x19e8c38, 0xc0005d4140, 0xc00033c480, 0x18d8f20, 0xc0004b60c0, 0x0, 0x0, 0x0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:347 +0x17f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc0003800e0, 0x19e8c38, 0xc0005d4140, 0xc000260070, 0x18d8f20, 0xc0004b60c0, 0xc000114248, 0x0, 0x0, 0x0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:624 +0x1cb
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc0003040a8, 0x19e8c38, 0xc0005d4140, 0xc0005d4180, 0xc0005d4140, 0x100b665, 0x187f4c0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:575 +0x43b
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadResource(0xc00029c0c0, 0x19e8ce0, 0xc0005d4140, 0xc0008a0240, 0xc00029c0c0, 0xc00074c9f0, 0xc0004d1ba0)
	github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:298 +0x105
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler(0x18b4b80, 0xc00029c0c0, 0x19e8ce0, 0xc00074c9f0, 0xc0008a01e0, 0x0, 0x19e8ce0, 0xc00074c9f0, 0xc000181500, 0x638)
	github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:344 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00022f6c0, 0x19efff8, 0xc000102f00, 0xc0005e6120, 0xc00010e7b0, 0x1e6de10, 0x0, 0x0, 0x0)
	google.golang.org/grpc@v1.37.0/server.go:1217 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc00022f6c0, 0x19efff8, 0xc000102f00, 0xc0005e6120, 0x0)
	google.golang.org/grpc@v1.37.0/server.go:1540 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc00030e1f0, 0xc00022f6c0, 0x19efff8, 0xc000102f00, 0xc0005e6120)
	google.golang.org/grpc@v1.37.0/server.go:878 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.37.0/server.go:876 +0x1fd
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x68 pc=0x16e6fdf]

goroutine 29 [running]:
github.com/alexashley/terraform-provider-rode/internal/provider.resourcePolicyRead(0x19e8ca8, 0xc000300d80, 0xc000195080, 0x18d8f20, 0xc0004b60c0, 0xc00026bbc0, 0xc0004d9908, 0x100df38)
	github.com/alexashley/terraform-provider-rode/internal/provider/resource_rode_policy.go:129 +0x1df
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc0003800e0, 0x19e8c38, 0xc000249980, 0xc000195080, 0x18d8f20, 0xc0004b60c0, 0x0, 0x0, 0x0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:347 +0x17f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc0003800e0, 0x19e8c38, 0xc000249980, 0xc000332230, 0x18d8f20, 0xc0004b60c0, 0xc0003082b0, 0x0, 0x0, 0x0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:624 +0x1cb
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc0003040a8, 0x19e8c38, 0xc000249980, 0xc0002499c0, 0xc000249980, 0x100b665, 0x187f4c0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:575 +0x43b
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadResource(0xc00029c0c0, 0x19e8ce0, 0xc000249980, 0xc000300a20, 0xc00029c0c0, 0xc000303620, 0xc0004d5ba0)
	github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:298 +0x105
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler(0x18b4b80, 0xc00029c0c0, 0x19e8ce0, 0xc000303620, 0xc0003009c0, 0x0, 0x19e8ce0, 0xc000303620, 0xc00014a000, 0xd1b)
	github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:344 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00022f6c0, 0x19efff8, 0xc000102f00, 0xc0004a8240, 0xc00010e7b0, 0x1e6de10, 0x0, 0x0, 0x0)
	google.golang.org/grpc@v1.37.0/server.go:1217 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc00022f6c0, 0x19efff8, 0xc000102f00, 0xc0004a8240, 0x0)
	google.golang.org/grpc@v1.37.0/server.go:1540 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc00030e1f0, 0xc00022f6c0, 0x19efff8, 0xc000102f00, 0xc0004a8240)
	google.golang.org/grpc@v1.37.0/server.go:878 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.37.0/server.go:876 +0x1fd
```

Nil dereference here:
https://github.com/rode/terraform-provider-rode/blob/e8c46bf0aeb324c65b74d992880219b7e0bf9bb0/internal/provider/resource_rode_policy.go#L129

There was a configuration issue with the client URL, but this should've propagated the gRPC error instead of crashing.

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # terraform-provider-rode
 
-[![test badge](https://github.com/alexashley/terraform-provider-rode/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/alexashley/terraform-provider-rode/actions/workflows/test.yaml?query=branch%3Amain)
+[![test badge](https://github.com/rode/terraform-provider-rode/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/rode/terraform-provider-rode/actions/workflows/test.yaml?query=branch%3Amain)
 
 
 A Terraform provider for [rode](https://github.com/rode/rode).

--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/alexashley/terraform-provider-rode
+module github.com/rode/terraform-provider-rode
 
 go 1.16
 

--- a/internal/provider/client.go
+++ b/internal/provider/client.go
@@ -15,11 +15,12 @@
 package provider
 
 import (
+	"log"
+	"sync"
+
 	"github.com/rode/rode/common"
 	"github.com/rode/rode/proto/v1alpha1"
 	"google.golang.org/grpc"
-	"log"
-	"sync"
 )
 
 type rodeClient struct {

--- a/internal/provider/client.go
+++ b/internal/provider/client.go
@@ -30,8 +30,8 @@ type rodeClient struct {
 	userAgent string
 }
 
+var clientInitErr error
 func (r *rodeClient) init() error {
-	var initErr error
 	r.Once.Do(func() {
 		log.Println("[DEBUG] Rode client init")
 		rode, err := common.NewRodeClient(
@@ -44,9 +44,9 @@ func (r *rodeClient) init() error {
 		}
 
 		r.RodeClient = rode
-		initErr = err
+		clientInitErr = err
 		log.Println("[DEBUG] Rode client init successful")
 	})
 
-	return initErr
+	return clientInitErr
 }

--- a/internal/provider/client.go
+++ b/internal/provider/client.go
@@ -31,6 +31,7 @@ type rodeClient struct {
 }
 
 var clientInitErr error
+
 func (r *rodeClient) init() error {
 	r.Once.Do(func() {
 		log.Println("[DEBUG] Rode client init")

--- a/internal/provider/provider.go
+++ b/internal/provider/provider.go
@@ -16,10 +16,11 @@ package provider
 
 import (
 	"context"
+	"log"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/rode/rode/common"
-	"log"
 )
 
 func init() {

--- a/internal/provider/provider_test.go
+++ b/internal/provider/provider_test.go
@@ -16,10 +16,11 @@ package provider
 
 import (
 	_ "embed"
-	"github.com/brianvoe/gofakeit/v6"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"os"
 	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 var (

--- a/internal/provider/resource_rode_policy.go
+++ b/internal/provider/resource_rode_policy.go
@@ -17,11 +17,12 @@ package provider
 import (
 	"context"
 	"fmt"
+	"log"
+
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/rode/rode/proto/v1alpha1"
-	"log"
 )
 
 func resourcePolicy() *schema.Resource {

--- a/internal/provider/resource_rode_policy_assignment.go
+++ b/internal/provider/resource_rode_policy_assignment.go
@@ -17,6 +17,9 @@ package provider
 import (
 	"context"
 	"fmt"
+	"log"
+	"strings"
+
 	"github.com/hashicorp/go-cty/cty"
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
@@ -24,8 +27,6 @@ import (
 	"github.com/rode/rode/proto/v1alpha1"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"log"
-	"strings"
 )
 
 var (

--- a/internal/provider/resource_rode_policy_assignment_test.go
+++ b/internal/provider/resource_rode_policy_assignment_test.go
@@ -18,15 +18,16 @@ import (
 	"context"
 	_ "embed"
 	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/rode/rode/proto/v1alpha1"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/proto"
-	"regexp"
-	"strings"
-	"testing"
 )
 
 func TestAccPolicyAssignment_basic(t *testing.T) {

--- a/internal/provider/resource_rode_policy_group.go
+++ b/internal/provider/resource_rode_policy_group.go
@@ -17,12 +17,13 @@ package provider
 import (
 	"context"
 	"fmt"
+	"log"
+	"regexp"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	"github.com/rode/rode/proto/v1alpha1"
-	"log"
-	"regexp"
 )
 
 var (

--- a/internal/provider/resource_rode_policy_group_test.go
+++ b/internal/provider/resource_rode_policy_group_test.go
@@ -17,12 +17,13 @@ package provider
 import (
 	"context"
 	"fmt"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-	"github.com/rode/rode/proto/v1alpha1"
 	"regexp"
 	"strings"
 	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/rode/rode/proto/v1alpha1"
 )
 
 func TestAccPolicyGroup_basic(t *testing.T) {

--- a/internal/provider/resource_rode_policy_test.go
+++ b/internal/provider/resource_rode_policy_test.go
@@ -17,12 +17,13 @@ package provider
 import (
 	"context"
 	"fmt"
+	"strings"
+	"testing"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/rode/rode/proto/v1alpha1"
 	"google.golang.org/protobuf/proto"
-	"strings"
-	"testing"
 )
 
 func TestAccPolicy_basic(t *testing.T) {

--- a/internal/provider/util.go
+++ b/internal/provider/util.go
@@ -16,11 +16,12 @@ package provider
 
 import (
 	"fmt"
-	"github.com/hashicorp/go-uuid"
-	"google.golang.org/protobuf/types/known/timestamppb"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/hashicorp/go-uuid"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 func formatProtoTimestamp(timestamp *timestamppb.Timestamp) string {

--- a/main.go
+++ b/main.go
@@ -17,9 +17,10 @@ package main
 import (
 	"context"
 	"flag"
-	"github.com/alexashley/terraform-provider-rode/internal/provider"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
+	"github.com/rode/terraform-provider-rode/internal/provider"
 )
 
 //go:generate go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs

